scripts/pkg.in

```diff
@@ -180,7 +180,11 @@ get_mirror_weight() {
 
 select_mirror() {
 	local current_mirror
-	current_mirror=$(grep -oE 'https?://[^ ]+' <(grep -m 1 -E '^[[:space:]]*deb[[:space:]]+' @TERMUX_PREFIX@/etc/apt/sources.list) || true)
+	if [ -f "@TERMUX_PREFIX@/etc/apt/sources.list" ]; then
+		current_mirror=$(grep -oE 'https?://[^ ]+' <(grep -m 1 -E '^[[:space:]]*deb[[:space:]]+' @TERMUX_PREFIX@/etc/apt/sources.list) || true)
+	elif [ -f "@TERMUX_PREFIX@/etc/apt/sources.list.d/main.sources" ]; then
```

Comment:
I've taken the liberty to swap the order of these in the `if` statement to prioritize `main.sources` since it should be the default repo entry moving forward, here as well as in `update_apt_cache()`.
This also matches the order in `has_repo()`